fingerprintdb_cuda.cpp

```diff
@@ -15,19 +17,25 @@
 using namespace gpusim;
 using std::vector;
 
-void FingerprintDB::search_cpu (const Fingerprint& query,
+void FingerprintDB::search_cpu (const Fingerprint& query, const QString& dbkey,
         std::vector<char*>& results_smiles,
         std::vector<char*>& results_ids,
         std::vector<float>& results_scores,
         unsigned int return_count,
         float similarity_cutoff) const
 {
```

Comment:
Sigh, everytime you make me look at this code, I'm going to ask for a results struct. And more consolidation between the cpu and gpu code...

Reply:
And every time I respond...